tests/ScreenshotTest.php

```diff
@@ -37,7 +37,7 @@ public function testTakeScreenshot(): void
 
         $screen = $client->takeScreenshot();
 
-        $this->assertIsString($screen);
+        $this->assertNotEmpty($screen);
```

Comment:
Why this change?

Reply:
`$client->takeScreenshot()` has a string return type, so `$screen`  is a string in any cases. We just expect a method call without exception.

The `assertNotEmpty` is simply here to avoid the test to be reported as risky. I could add a `expectNotToPerformAssertions()` instead but it seemed ok to me to assert that a screenshot isn't an empty string.